lib/metadata/models/object/ObjectMD.js

```diff
@@ -571,6 +577,26 @@ module.exports = class ObjectMD {
     }
 
     /**
+     * Set replication information
+     *
+     * @param {object} replicationInfo - replication information object
+     * @return {ObjectMD} itself
+     */
+    setReplicationInfo(replicationInfo) {
```

Comment:
If `storageClass` is not included in the rule, I assume we want to keep `replicationInfo.storageClass` as `''` in the object MD.